src/fsharp/FSStrings.resx

```diff
@@ -1036,10 +1036,10 @@
     <value>Interface implementations should be given on the initial declaration of a type.</value>
   </data>
   <data name="UnresolvedReferenceNoRange" xml:space="preserve">
-    <value>A required assembly reference is missing. You must add a reference to assembly '{0}'.</value>
+    <value>The type '{0}' is required here and is unavailable. You must add a reference to assembly '{1}'.</value>
   </data>
   <data name="UnresolvedPathReferenceNoRange" xml:space="preserve">
-    <value>The type referenced through '{0}' is defined in an assembly that is not referenced. You must add a reference to assembly '{1}'.</value>
+    <value>A reference to the type '{0}' in assembly '{1}' was found, but the type could not be found in that assembly.</value>
```

Comment:
Has this broken the codefixer that looks for this compile error to automatically add references?